test/Microsoft.AspNet.FileProviders.Physical.Tests/PhysicalFileProviderTests.cs

```diff
@@ -669,7 +680,7 @@ public void Tokens_With_Forward_And_Backward_Slash()
         }
 
         [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [OSSkipCondition(OperatingSystems.Linux, SkipReason = "Race condition in inotify API makes this very flaky.")]
```

Comment:
Triage with @muratg to decide if this is an scenario we should address in production, or if the scenario is un-important and we should re-write the test.